app/templates/components/widgets/forms/time-picker.hbs

```diff
@@ -1,4 +1,4 @@
 {{#if icon}}
   <i class="time icon"></i>
 {{/if}}
-{{input type='text' value=value placeholder=placeholder name=name}}
+{{input type='text' value=value placeholder=placeholder id=inputId name=name}}
```

Comment:
Because we're customising the ID. We use ID instead of name to refer to it here.